.pre-commit-config.yaml

```
@@ -31,7 +31,7 @@ repos:
   rev: v1.17.0
   hooks:
   - id: mypy
-    exclude: tests/integration/.*/.*|tests/integration/quick_check_numpy.py|tests/unit/test_wheel_abi.py|scripts/calculate_symbol_versions.py
+    exclude: tests/integration/.*/.*|tests/integration/.*|tests/integration/quick_check_numpy.py|tests/unit/test_wheel_abi.py|scripts/calculate_symbol_versions.py
```

Please restore mypy check, it should be okay once ubuntu & centos are checked separately.
```suggestion
    exclude: tests/integration/.*/.*|tests/integration/quick_check_numpy.py|tests/unit/test_wheel_abi.py|scripts/calculate_symbol_versions.py
```